ocrd_network/setup.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from setuptools import setup
+from ocrd_utils import VERSION
+
+install_requires = open('requirements.txt').read().split('\n')
+install_requires.append('ocrd_utils == %s' % VERSION)
+install_requires.append('ocrd_validators == %s' % VERSION)
+
+setup(
+    name='ocrd_network',
+    version=VERSION,
+    description='OCR-D framework - network',
+    long_description=open('README.md').read(),
+    long_description_content_type='text/markdown',
+    author='Konstantin Baierer',
+    author_email='unixprog@gmail.com',
+    url='https://github.com/OCR-D/core',
+    license='Apache License 2.0',
+    install_requires=install_requires,
+    packages=[
+        'ocrd_network',
+        'ocrd_network.models',
+        'ocrd_network.rabbitmq_utils'
+    ],
+    package_data={
+        '': ['*.yml', '*.xsd']
```

Comment:
Necessary? I.e. is there any non-python data to be packaged?

Reply:
Not necessary, will remove it.